packages/lib/src/toolbar/controls/ExportMenu.tsx

```diff
@@ -0,0 +1,60 @@
+import { isDefined } from '@h5web/shared';
+import { Button, Wrapper, Menu } from 'react-aria-menubutton';
+import { FiDownload } from 'react-icons/fi';
+import { MdArrowDropDown } from 'react-icons/md';
+
+import styles from './Selector/Selector.module.css';
+
+interface Props<F extends string> {
+  formats: F[];
+  isSlice: boolean;
+  getFormatURL: (format: F) => string | undefined; // `undefined` if format is not supported
+}
+
+function ExportMenu<F extends string>(props: Props<F>) {
+  const { formats, isSlice, getFormatURL } = props;
+
+  const urls = getFormatURL ? formats.map(getFormatURL) : [];
+  const hasSupportedFormats = urls.some(isDefined);
+
+  return (
+    <Wrapper className={styles.wrapper}>
+      <Button
+        className={styles.btn}
+        tag="button"
+        disabled={!hasSupportedFormats}
```

Comment:
I now disable the menu if the provider doesn't implement `getExportURL` or doesn't support any of the given formats. This avoids ending up with a `Separator` that has nothing to separate.